src/app/SharpRaven/Utilities/Serializer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+
+namespace SharpRaven.Utilities
+{
+    public static class Serializer
+    {
+        public static byte[] SerialiseBytes(this object obj)
```

Comment:
Since this is only used for `JsonPacket` and it already does serialization to JSON, I don't think we need a separate class with a general use extension method for `System.Object`.